readme.md

```diff
@@ -150,6 +150,10 @@ Adds the provided alias info into webpack's alias section. Pass an object litera
 
 Adds the provided resolve info into webpack's resolve section. Pass an object literal with as many entries as you'd like, and the whole object will be merged in.
 
+### addWebpackExternals(deps)
+
+Add external dependencies, useful when trying to offload libs to CDN
```

Comment:
May you provide a short example to explain how to use this?

Reply:
Here you go 

https://webpack.js.org/configuration/externals/
https://github.com/facebook/create-react-app/issues/2758

Should I add them to the documents?

Reply:
Ideally the example should be written directly in the document

Reply:
Also firebase js sdk https://firebase.google.com/docs/web/setup#reserved-urls with externals we can import firebase in js file also using the js firebase provided